mithril-aggregator/src/file_uploaders/gcp_uploader.rs

```diff
@@ -85,10 +86,10 @@ pub struct GcpBackendUploader {
 impl GcpBackendUploader {
     /// GcpBackendUploader factory
     pub fn try_new(bucket: String, use_cdn_domain: bool, logger: Logger) -> StdResult<Self> {
-        if env::var("GOOGLE_APPLICATION_CREDENTIALS_JSON").is_err() {
-            return Err(anyhow!(
-                "Missing GOOGLE_APPLICATION_CREDENTIALS_JSON environment variable".to_string()
-            ));
+        if env::var(DEFAULT_GCP_CREDENTIALS_JSON_ENV_VAR).is_err() {
```

Comment:
We need 2 different variables if we want to use 2 different service accounts:
```suggestion
        if env::var(DEFAULT_GCP_CREDENTIALS_CLOUD_STORAGE_JSON_ENV_VAR).is_err() {
```